### PR TITLE
add epazote health check file [OFWEB-1018]

### PR DESCRIPTION
**Pull request by SinisaG**
https://motainteam.atlassian.net/browse/OFWEB-1018

### CODE REVIEW

File: service/phantomJs.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ phantomJS.screenshot = function (req, res) {
         return;
     }
     
-    phantom.create(function (ph) {
+    phantom.create('--web-security=no', function (ph) {
```
Comment (SinisaG) [inline service/phantomJs.js]: This switch is needed for cross domain requests.

File: epazote.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+services:
+ peregrine:
+  url: http://127.0.0.1:3000/api/screenshot/https%3A%2F%2Fperegrine-api.onefootball.com%2Fphotobooth%2Fmatch%2F9%2F432927%3Flocale%3Den-US%26period%3DPreMatch%26variable%3Dg-2
+  seconds: 1800
```
Comment (SinisaG) [inline epazote.yml]: I've put it on 30 minutes to prevent too much pressure on api.

**Comment by maximevaly** [comment]
:+1: looks good, should we also add a short comment at the beginning of the file to explain its use?

**Comment by NKjoep** [comment]
:+1: 

a short comment is welcome :)

**Comment by SinisaG** [comment]
Added. :)

**Pull request merged**